lib/matplotlib/backends/backend_qt5.py

```diff
@@ -678,7 +675,7 @@ def destroy(self, *args):
         self.window.close()
 
     def get_window_title(self):
-        return six.text_type(self.window.windowTitle())
+        return self.window.windowTitle()
```

Comment:
PyQt type <-> python type:

Should this still be wrapped in a `str()`? Otherwise we'll return a `QString()`, which I don't think we want to leak out of the backend API?

Reply:
this actually returns a str (automatic conversion, see http://pyqt.sourceforge.net/Docs/PyQt5/gotchas.html#python-strings-qt-strings-and-unicode), try yourself `plt.gcf().canvas.parent().windowTitle()`
same comment applies throughout

Reply:
It works :smile:. Not sure I understand why

The docs on default conversions do only speak of "If Qt expects ...". For me that sounded like parameter conversion. Somehow the return type is converted as well?!? So I never would have the chance of getting the actual QString here? I should really learn about the changes in PyQt5.

Reply:
See also http://pyqt.sourceforge.net/Docs/PyQt4/incompatible_apis.html
We basically don't support the qt4v1 api (too many places where we already assume that we get native strings out of qt methods).